### PR TITLE
Supports aomap

### PR DESCRIPTION
**Pull request by panxinmiao**
Modified aomap to use the second set of texture coordinates and improved the support for aomap in MeshBasicMaterial, MeshPhongMaterial, and MeshStandardMaterial.

image

### CODE REVIEW

File: examples/feature_demo/aomap.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+"""
+Ambient occlusion
+========
```
Comment (almarklein) [inline examples/feature_demo/aomap.py:3]: Title underline must be the same length as the title (this makes Sphinx complain and causes the doc build to fail 😄).
```suggestion
Ambient occlusion
=================
```

File: pygfx/materials/_mesh.py [hunk 2]
```diff
@@ -356,6 +363,28 @@ def light_map_intensity(self, value):
...
+
+    @ao_map.setter
+    def ao_map(self, map):
+        if map is not None and not isinstance(map, Texture):
+            raise ValueError(f"ao_map must be a Texture or None, received: {type(map)}")
+        self._store.ao_map = map
+
+    @property
+    def ao_map_intensity(self):
+        """Intensity of the ambient occlusion effect. Default is 1.0 Zero is no occlusion effect."""
```
Comment (almarklein) [inline pygfx/materials/_mesh.py:380]: Missing comma/dot.
```suggestion
        """Intensity of the ambient occlusion effect. Default is 1.0, zero is no occlusion effect."""
```

**Review by almarklein — COMMENTED** [review]
Very nice work again! I have just two small textual suggestions.

File: examples/feature_demo/aomap.py (hunk 1 above)
Comment (panxinmiao) [reply examples/feature_demo/aomap.py:3]: Ah, I finally know the reason.

**Review by almarklein — APPROVED** [review]

**Review by Korijn — APPROVED** [review]
Amazing 🤩

**Pull request merged**